cmd/nitro/nitro.go

```diff
@@ -934,3 +941,80 @@ func testUpdateTxIndex(chainDb ethdb.Database, chainConfig *params.ChainConfig)
 	}
 	log.Info("Tx lookup entries written")
 }
+
+type LiveNodeConfig struct {
+	mutex  sync.RWMutex
+	args   []string
+	config *NodeConfig
+}
+
+func (c *LiveNodeConfig) Get() *NodeConfig {
+	c.mutex.RLock()
+	defer c.mutex.RUnlock()
+	return c.config
+}
+
+func (c *LiveNodeConfig) set(config *NodeConfig) error {
+	c.mutex.Lock()
+	defer c.mutex.Unlock()
+
+	newConfig := *c.config
+
+	newConfig.Node.Sequencer.MaxBlockSpeed = config.Node.Sequencer.MaxBlockSpeed
+
+	err := initLog(config.LogType, log.Lvl(config.LogLevel))
+	if err != nil {
```

Comment:
Not that you have to or that it's always cleanest to do this, but you can often tuck short `err` expressions inside an if like so
```go
if err := initLog(config.LogType, log.Lvl(config.LogLevel)); err != nil {
    return err
}